### PR TITLE
Clear azure rm context in initialize azure

### PR DESCRIPTION
**Pull request by SumiranAgg**

### CODE REVIEW

File: Tasks/Common/VstsAzureHelpers_/InitializeFunctions.ps1 [hunk 1]
```diff
@@ -148,7 +148,11 @@ function Initialize-AzureSubscription {
         } else {
             # Else, this is AzureRM.
             try {
-                if (Get-Command -Name "Add-AzureRmAccount" -ErrorAction "SilentlyContinue") {
+                if (Get-Command -Name "Add-AzureRmAccount" -ErrorAction "SilentlyContinue") {                    
+                    Write-Host "##[command]Clear-AzureRmContext -Scope Process"
+                    $null = Clear-AzureRmContext -Scope Process
+                    Write-Host "##[command]Clear-AzureRmContext -Scope CurrentUser -Force -ErrorAction SilentlyContinue"
+                    $null = Clear-AzureRmContext -Scope CurrentUser -Force -ErrorAction SilentlyContinue
```
[the remark this answers is not in the document]
Comment (SumiranAgg) [reply Tasks/Common/VstsAzureHelpers_/InitializeFunctions.ps1]: The recommendation was to Select-AzureRmContext, but that will not resolve all issues. We are still under discussion with Azure PS team. In the meanwhile, this is the fix we have recognized.

File: Tasks/Common/VstsAzureHelpers_/InitializeFunctions.ps1 [hunk 2]
```diff
@@ -148,7 +148,11 @@ function Initialize-AzureSubscription {
         } else {
             # Else, this is AzureRM.
             try {
-                if (Get-Command -Name "Add-AzureRmAccount" -ErrorAction "SilentlyContinue") {
+                if (Get-Command -Name "Add-AzureRmAccount" -ErrorAction "SilentlyContinue") {                    
+                    Write-Host "##[command]Clear-AzureRmContext -Scope Process"
+                    $null = Clear-AzureRmContext -Scope Process
```
[the remark this answers is not in the document]
Comment (SumiranAgg) [reply Tasks/Common/VstsAzureHelpers_/InitializeFunctions.ps1]: Yes, the command is available from version 4.

[the remark this answers is not in the document]
Comment (Ajay-MS) [reply Tasks/Common/VstsAzureHelpers_/InitializeFunctions.ps1]: Can you please check if this cmdlet exists.  I think that way it won't even break for customers on version less that 4 too.

[the remark this answers is not in the document]
Comment (Ajay-MS) [reply Tasks/Common/VstsAzureHelpers_/InitializeFunctions.ps1]: Can you please check if this cmdlet exists. I think that way it won't even break for customers on version less than 4 too.

[the remark this answers is not in the document]
Comment (Ajay-MS) [reply Tasks/Common/VstsAzureHelpers_/InitializeFunctions.ps1]: Should we move this cmdlet outside of the if condition.

[the remark this answers is not in the document]
Comment (SumiranAgg) [reply Tasks/Common/VstsAzureHelpers_/InitializeFunctions.ps1]: AS per offline discussion made changes

**Pull request merged**